plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/models/IRunningPod.java

```diff
@@ -0,0 +1,16 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc. Distributed under license by Red Hat, Inc.
```

Comment:
i think it's better to have 2016 here and everywhere now as it is done in RunningPodAdapterFactory